app/controllers/api/v1/hosts_controller.rb

```diff
@@ -45,7 +48,10 @@ def show
         param :owner_id, :number
         param :puppet_ca_proxy_id, :number
         param :image_id, :number
-        param :host_parameters_attributes, Array
+          param :host_parameters_attributes, Array, :desc => N_("Host's parameters (array or indexed hash)") do
```

Comment:
indentation's off here, and there's no need for translations in the v1 controller